packages/launchpad/src/App.vue

```diff
@@ -28,10 +28,14 @@ query AppQuery {
  */
 const query = useQuery({ 
   query: AppQueryDocument,
-  requestPolicy: 'network-only'
+  requestPolicy: 'cache-and-network'
```

Comment:
I think this was intentionally `network-only` since we are using it for polling. This polling hack probably needs to die anyway, but if we are doing a polling query it should always make a network request, and not use any caching - at least that's my understanding and logic.

Reply:
I think that the `cache-and-network` should also work, b/c the value won't first be in the cache to begin with so it'll hit the network anyway

Reply:
But agree we need to get rid of this hack